spec/fixtures/api/native-window-open.html

```diff
@@ -0,0 +1,30 @@
+<html>
+<body>
+<script type="text/javascript" charset="utf-8">
+  const {ipcRenderer} = require("electron");
+  const tests = {
+    'blank': () => {
```

Comment:
I think it might be nice to split up this file into 3 separate files, one for each case being tested. Then we don't risk one test case possibly influencing another since they will each be self-contained.